cfbs/commands.py

```diff
@@ -392,6 +405,20 @@ def _get_module_by_name(name) -> dict:
                 return module
         return None
 
+    def _build_user_prompt(module, dependencies):
```

Comment:
This should probably be more descriptive, given that it is in the global scope. E.g., `_build_remove_module_user_prompt()`